roles/confluent.test/molecule/mtls-custombundle-rhel/molecule.yml

```diff
@@ -128,6 +128,9 @@ provisioner:
         ssl_enabled: true
         ssl_mutual_auth_enabled: true
 
+        # cert creation script does not use fqdn, which is a requirement for zk ssl
```

Comment:
Should we modify this script to support FQDN?  How are we testing this?

Reply:
the certs-create script now adds the fqdn, which most tls scenarios use
create_ca_bundle script focuses less on the hostnames and more on having intermediate cas/ signing certs by diff cas
So i think testing coverage is good